sonar-ws/src/main/gen-java/org/sonarqube/ws/QualityProfiles.java

```diff
@@ -0,0 +1,2483 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
+// source: ws-qualityprofiles.proto
+
+package org.sonarqube.ws;
+
+public final class QualityProfiles {
+  private QualityProfiles() {}
+  public static void registerAllExtensions(
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1186)